sigs/sig-serverless.md

```diff
@@ -0,0 +1,139 @@
+## CNCF Serverless SIG Charter
```

Comment:
Nope we didn't because, as you said, we're just morphing from a WG. The WG did review/comment on it though. We kind of assumed that we'd deal with edits via the normal PR process.